### PR TITLE
docs(airbnb): add map to the results

### PR DESCRIPTION
**Pull request by vvo**
map

### CODE REVIEW

**Comment by vvo** [comment]
cc @redox finally!

File: docs/examples/airbnb/search.js [hunk 1]
```diff
@@ -68,7 +70,13 @@ search.addWidget(
...
   instantsearch.widgets.rangeSlider({
     container: '#price',
-    facetName: 'price'
+    attributeName: 'price'
+  })
+);
+
+search.addWidget(
+  instantsearchGoogleMaps({
+    container: document.querySelector('#map')
```
Comment (vvo) [inline docs/examples/airbnb/search.js]: 3 lines: widgets!

giphy 14

**Comment by maxiloc** [comment]
:+1: Looks really nice

**Comment by vvo** [comment]
We will need some tuning so let's try it once merged and see what we need to do:
- provide an option to refine on zoom/move (right now activated by default, should not)

File: docs/examples/airbnb/search.js [hunk 2]
```diff
@@ -68,7 +70,13 @@ search.addWidget(
 search.addWidget(
   instantsearch.widgets.rangeSlider({
     container: '#price',
-    facetName: 'price'
+    attributeName: 'price'
+  })
+);
+
+search.addWidget(
+  instantsearchGoogleMaps({
```
Comment (redox) [inline docs/examples/airbnb/search.js]: `s/instantsearchGoogleMaps/instantsearch.widgets.googleMaps/g` ?

Comment (vvo) [inline docs/examples/airbnb/search.js]: yes, will do

Comment (vvo) [inline docs/examples/airbnb/search.js]: fixed

**Comment by Jerskouille** [comment]
Damn, it's so fast!

**Pull request merged**